src/View/Requirements_Backend.php

```diff
@@ -1004,13 +1005,23 @@ public function add_i18n_javascript($langDir, $return = false)
 
         $files = array();
         $candidates = array(
-            'en.js',
-            'en_US.js',
-            i18n::getData()->langFromLocale(i18n::config()->get('default_locale')) . '.js',
-            i18n::config()->get('default_locale') . '.js',
-            i18n::getData()->langFromLocale(i18n::get_locale()) . '.js',
-            i18n::get_locale() . '.js',
+            'en',
+            'en_US',
+            i18n::getData()->langFromLocale(i18n::config()->get('default_locale')),
+            i18n::config()->get('default_locale'),
+            i18n::getData()->langFromLocale(i18n::get_locale()),
+            i18n::get_locale(),
+            strtolower(DBField::create_field('Locale', i18n::get_locale())->RFC1766()),
```

Comment:
Should `strtolower` be encapsulated inside `RFC1766`?

Reply:
It's not clear. On the [W3C site](https://www.w3.org/International/articles/language-tags/) they code the region in upper-case, but Microsoft doesn't in their doc (https://docs.microsoft.com/en-us/windows/desktop/wmformat/language-strings).

The region bit should confirm to the RFC3066 standard. The IETF site says it doesn't matter. https://tools.ietf.org/html/rfc3066#section-2.1

In any case, changing to lower case everywhere could break other things, so we're probably best not to touch it right now.